### DISCUSSION TITLE
How sync time of paxcounter with heltec lora32 v2

### DISCUSSION THREAD

**Discussion by HelloWar75** [cyberman54/ESP32-Paxcounter#725] [Q&A] [answered]
How sync time of paxcounter with heltec lora32 v2, the lorawan gateway not have uplink or need the uplink in my gateway ?

**Comment by cyberman54** [comment, marked as answer]
You either need a LORAWAN network server supporting TimeReq (LORAWAN >= 1.0.3), or you can build your own timeserver using the example in the repo. Both methods won't work if your LORAWAN gateway has no uplink. But gateways without uplink to a LORAWAN network server won't do anything.

**Reply by HelloWar75** [reply]
I'm setting up my gateway following this tutorial would you know if you would have uplink?

https://www.hackster.io/ChrisSamuelson/lora-raspberry-pi-single-channel-gateway-cheap-d57d36

**Reply by cyberman54** [reply]
Please ask in the hackster.io forum, since this is not a paxcounter related topic.
Note that single channel gateways are not LORAWAN compliant, what means paxcounter (and most other LORAWAN nodes) can't work with it.
